app/models/package_manager/packagist.rb

```diff
@@ -27,7 +27,8 @@ def self.recent_names
     end
 
     def self.project(name)
-      get("https://packagist.org/packages/#{name}.json")["package"]
+      get("https://packagist.org/packages/#{name}.json")
```

Comment:
Should this explode tho, and not silently? Maybe just a better error?

Reply:
For package updates, our convention is to [move on silently](https://github.com/librariesio/libraries.io/blob/main/app/models/package_manager/base.rb#L97) if the project is blank. I think it makes sense to just return a `blank?` value from this if the project is 404'ing, and let the "check_project_status" code mark it as "Removed" or whatever later on. I made a similar change [to Sublime](https://github.com/librariesio/libraries.io/blob/main/app/models/package_manager/sublime.rb#L18-L22) recently too.